migration/migrations/001_initial.py

```diff
@@ -0,0 +1 @@
+print("Initial Migration")
```

Comment:
great start!
you'll certainly want to capture the `revision` string and `down_revision` (none in the initial case, only), for the migration framework to determine the order, etc.